src/main/java/seedu/address/model/widget/WidgetModel.java

```diff
@@ -21,14 +20,14 @@ static WidgetModel initWidget() {
     /**
      * Sets the content to be displayed in the view box to be of that content.
      *
-     * @param content
+     * @param client
```

Comment:
probably can remove the `@param` since there's no description, and adding a description would be redundant